### PR TITLE
Fixed #2061 - SVG in hero animation

### LINKED ISSUE #2061: SVG in hero animation - bug + bulb
During the process, we mixed up the naming of 2 SVG's. 

This is why we need to change [`svg-bulb_nobug.svg`](https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/static/img/svg/icons/svg-bulb_nobug.svg) and [`svg-bulb.svg`](https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/static/img/svg/icons/svg-bulb.svg).

- [x] change `svg-bulb_nobug.svg` to `svg-bulb.svg`
- [x] change `svg-bulb.svg` to `svg-bug-animated.svg`
- [x] run `npm run build:svg` in your terminal
- [x] copy [sprite](https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/static/img/svg/sprite.svg) in [html include](https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/templates/shared/svg-icons.html) (Make sure, you keep the `is-hidden` class)
- [x] change SVG names / IDs in CSS files (search & replace for `svg-bulb_nobug.svg` and `svg-bulb.svg`)
- [x] test animation in index page

### PR DESCRIPTION
**Pull request by mpaarating**
r? @zoepage 

This renames some the svg files that had been previously mistakenly
renamed and replaces the corrected names throughout the project.

### CODE REVIEW

File: webcompat/templates/home-page/hero.html [hunk 1]
```diff
@@ -12,12 +12,12 @@ <h1 class="hero-claim highlight js-hero-title">Bug reporting <br> for the web.</
     <aside class="hero-illustration">
       <svg class="hero-illustration-bulb" role="presentation" viewBox="0 0 160 447" aria-role="hide">
         <title>bulb_nobug</title>
```
Comment (zoepage) [inline webcompat/templates/home-page/hero.html]: could you please change the title here too?

File: webcompat/templates/home-page/hero.html [hunk 2]
```diff
@@ -12,12 +12,12 @@ <h1 class="hero-claim highlight js-hero-title">Bug reporting <br> for the web.</
...
-        <use href="#svg-bulb_nobug">bulb</use>
+        <use href="#svg-bulb">bulb</use>
       </svg>
 
       <svg class="hero-illustration-bug bug-animated" role="presentation" viewBox="0 0 47 65" aria-role="hide">
         <title>bulb</title>
```
Comment (zoepage) [inline webcompat/templates/home-page/hero.html]: could you please change the title here too?

**Comment by mpaarating** [comment]
Should be good now!

**Comment by zoepage** [comment]
I think, this should be ok :)

For the future, maybe this is interesting for you. For changing commit messages, you have two options. 

1. You can run `git commit --amend` and change your last commit message.
https://www.atlassian.com/git/tutorials/rewriting-history

OR

2. You can run `git reset --soft HEAD^` (for one step back) or `git rebase --soft commitNumber` for a revert to a specific commit, without losing your changes. Then you can add + commit again as you are used to it. https://git-scm.com/docs/git-reset

Anyway, thanks!

**Pull request merged**